### ISSUE TITLE
Second iteration of community training: July - Sep

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/open-community-building#75]
### Summary Sentence

We have recently finished onboarding all the expected arrivals in CM team, and additionally have onboarded Clau from Turing Commons. To make sure that we all have similar basic of community management, I will be repeating a team workshop in July/August. This is similar to what was delivered in December and January that most of you have already attended (no need to repeat unless you would like to). 

### What needs to be done?

In this workshop I will be discussing:
- :construction: Community foundation: What is your community’s story, who started it, what was the reason/purpose and where we want to take it?
- :abc: Community of Practice basics: Purpose and outcomes, stakeholder mapping, roles and responsibility documentation and communication channels
- :wave::skin-tone-4: Community engagement: mountain of engagement, personas and pathways, incentives and value-exchange
- :scroll: Community charter: vision, mission, milestones, roadmap, ways of working
- [What else would you want to learn/discuss?]

I would like to make sure that we are embedding the practices in the work the TPS promises, thinking about building connection with each other and sharing resources around these topics. 

### Who can help?

- I hope to run this workshop in short chunks (1 hour) in the alternate week when the team meeting is not scheduled. I would love to have the not so new CMs to deliver 1-2 topics within these modules of your choice. We have materials that you can adapt to include work from your community. Please let me know if you would be interested (I really hope you do!)
- If you have suggestions on additional topics, please also reply below, or directly comment on this issue.

---

### Update after the issue was opened

[Add details]

**Comment by malvikasharan** [comment]
We are going with 13:00 - 14:00 London time, every second Thursday starting from 14 July.

**Comment by dingaaling** [comment]
@malvikasharan - would RAMs be able to join this, by chance? A bit of a late in the game request...

**Comment by malvikasharan** [comment]
@dingaaling of course. Can you let me know who would like to join? I can share the cal invite with you.

**Comment by harisood** [comment]
Would be v interested in joining!

**Comment by aldenc** [comment]
I'm not sure if they time will always work for me but please include me on the calendar invite!
